imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSDLayerInfo.java

```diff
@@ -57,14 +57,22 @@
     private String unicodeLayerName;
     private int layerId;
 
+    static private Stack<List<PSDLayerInfo>> groupStack = new Stack<>();
+    static private List<PSDLayerInfo> group = null;
+    Integer groupLayerId;
+    Boolean isGroup = false;
+    Boolean isSectionDivider = false;
```

Comment:
Is there a reason these are boxed types? I usually prefer using primitives when possible.